weatherAplication/Model/StorageService.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+final class StorageService {
+    let lastSearchesKey = "lastSearchesKey"
```

Comment:
taki klucz powinien być prywatny

Reply:
I w sumie jak w nazwie stałej jest słowo Key to może już w samej wartości nie dodawaj Key bo wtedy mamy trochę masło maślane - lastSearchesKey = "lastSearches"